bears/python/PyImportSortBear.py

```diff
@@ -182,7 +184,18 @@ def run(self, filename, file,
 
         if new_file != tuple(file):
             diff = Diff.from_string_arrays(file, new_file)
+            all_imports = ''
+            for line_nr in sorted(diff._changes):
+                linediff = diff._file[max(line_nr - 1, 0)]
+                list_of_imports = re.split(', +',
+                                           linediff.rstrip())
+                if list_of_imports[0].startswith('import'):
+                    list_of_imports[0] = list_of_imports[0].lstrip('import')
```

Comment:
`lstrip` doesn't work like this, for example this string
`importimoopX`
would expand to
`X`
as `lstrip` strips **each character** away, not a complete string in the given order. You can use `str.replace` or if you need additional whitespace stripping, you can use regexes and their replacement functions